templates/todo/projects/nodejs-mongo/.repo/bicep/repo.yaml

```diff
@@ -31,6 +31,11 @@ repo:
         patterns:
           - "**/azure.@(yml|yaml)"
 
+      - from: "PLACEHOLDERIACTOOLS"
```

Comment:
This is a cute trick!  Love how we were able to share so much between the README's with this.

Reply:
Yes, loving the unintended usage of the repoman rewrite rules 🤣